app/javascript/store/modules/admin/actions.js

```diff
@@ -6,8 +6,11 @@ import {
 
 export default {
     [UPDATE_DEFAULT_TEAM](
-      {}, { teamId, organization }) {
+      {}, { teamId, organization, froggoTeam }) {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 4 spaces but found 6   indent